tests/snuba/api/endpoints/test_organization_events_span_indexed.py

```diff
@@ -547,11 +547,12 @@ def test_simple(self):
         )
         response = self.do_request(
             {
-                "field": ["span.status", "description", "count()"],
+                "field": ["description", "count()"],
                 "query": "",
                 "orderby": "description",
                 "project": self.project.id,
                 "dataset": self.dataset,
+                "spans_rpc": 1,
```

Comment:
This test is failing with
```  File "/Users/wmak/development/sentry/src/sentry/utils/snuba_rpc.py", line 73, i
n rpc
    resp.ParseFromString(http_resp.data)
google.protobuf.message.DecodeError: Error parsing message
```

which is from the response being
```b'<!doctype html>\n<html lang=en>\n<title>405 Method Not Allowed</title>\n<h1>Met
hod Not Allowed</h1>\n<p>The method is not allowed for the requested URL.</p>\n'
```

Not sure why the method isn't allowed yet